openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -406,12 +406,12 @@ There are two possible ways to request issuance of a specific Credential type in
 
 ### Request Issuance of a Certain Credential Type using `authorization_details` Parameter {#authorization-details}
 
-The request parameter `authorization_details` defined in Section 2 of [@!RFC9396] MUST be used to convey the details about the Credentials the Wallet wants to obtain. This specification introduces a new authorization details type `openid_credential` and defines the following elements to be used with this authorization details type:
+The request parameter `authorization_details` defined in Section 2 of [@!RFC9396] MUST be used to convey the details about the Credentials the Wallet wants to obtain. This specification introduces a new authorization details type `openid_credential` and defines the following parameters to be used with this authorization details type:
 
 * `type` REQUIRED. JSON string that determines the authorization details type. MUST be set to `openid_credential` for the purpose of this specification.
```

Comment:
```suggestion
* `type` REQUIRED. String that determines the authorization details type. MUST be set to `openid_credential` for the purpose of this specification.
```

Reply:
we used to say `string` and i don't remember who, but someone said it should be `JSON string` and we changed... If we change to string here, we should change throughout the spec..

Reply:
It doesn't make sense to say JSON string, as the data structure is a string and the encoding is JSON, but the JSON encoding is applied to the whole object, not just this data field. There is no need to say JSON here and it may even be misinterpreted as implying a JSON encoding for this string, like so:
```json
{
  "type": "\"example\""
}
```

Reply:
good point on potential misinterpretation.